server/graphql-schema.js

```diff
@@ -740,6 +740,30 @@ const graphqlSchema = gql`
         conflictingResults: [PopulatedData]!
     }
 
+    type Issue {
+        """
+        GitHub username of the issue creator.
+        """
+        author: String!
+        """
+        Link to the GitHub issue's first comment.
+        """
+        link: String!
+        """
+        Indicates the type of issue. 'changes-requested' or 'feedback'.
+        'feedback' is the default type.
+        """
+        type: String!
+        """
+        Indicates if the issue is currently open on GitHub.
+        """
+        isOpen: Boolean!
+        """
+        Test Number the issue was raised for.
+        """
+        testNumber: Int!
```

Comment:
Is this the same as the rowNumber?

Reply:
This is slightly different to `rowNumber` in that `rowNumber` refers to the sequence number of the test from the original set of tests (the .csv file). Whereas the `testNumber` here is referring to the sequence number in the set of tests grouped by the AT if that helps to clear up anything here.

Reply:
Ah, that's a very handy explanation. In that case I would probably vote for a more wordy variable name or at the very least putting an explanation in the documentation string. Maybe a name like `testNumberFilteredByAt`.